inc/Editor/DataBinding/BlockBindings.php

```diff
@@ -198,6 +198,32 @@ private static function execute_queries( array $block_context, array $source_arg
 		}
 	}
 
+	public static function should_render_empty_result( WP_Block $block ): bool {
```

Comment:
This has been done this way so that the calls are centralized to this class just like the pagination block.